src/pages/edit_resource/ResourceFormActions.tsx

```diff
@@ -74,34 +87,43 @@ const ResourceFormAction: FC<ResourceFormActionProps> = ({ activeStep, allChange
     }
   };
 
+  const handleLeaveForm = async () => {
+    history.push('resources/user/current');
+  };
+
   return (
     <StyledSchemaPart>
-      <StyledButtonWrapper>
-        {activeStep > 0 ? (
-          <UpperCaseButton variant="outlined" disabled={activeStep === 0} onClick={handleBack}>
-            <StyledArrowBackIcon /> <StyledButtonText>{t(getStepLabel(activeStep - 1))}</StyledButtonText>
-          </UpperCaseButton>
-        ) : (
-          <span />
-        )}
-        <StyledRightSideButtonWrapper>
-          <StyledFormStatusWrapper>
-            {!allChangesSaved && <CircularProgress size="1rem" />}
-            {allChangesSaved && !dirty && <Typography variant={'body1'}>{t('common.all_changes_saved')}</Typography>}
-          </StyledFormStatusWrapper>
-          {activeStep === ResourceFormStep.Preview ? (
-            <UpperCaseButton variant="contained" color="primary" onClick={handlePublishResource} disabled={!isValid}>
-              {t('common.publish')}
-            </UpperCaseButton>
-          ) : (
-            <UpperCaseButton variant="contained" color="primary" onClick={handleNext}>
-              <StyledButtonText>{t(getStepLabel(activeStep + 1))}</StyledButtonText>
-              <StyledArrowForwardIcon />
-            </UpperCaseButton>
-          )}
-        </StyledRightSideButtonWrapper>
-      </StyledButtonWrapper>
-      {publishResourceError && <ErrorBanner />}
+      <StyledContentWrapperMedium>
+        <StyledButtonWrapper>
+          <div>
+            {activeStep > 0 ? (
+              <UpperCaseButton variant="outlined" disabled={activeStep === 0} onClick={handleBack}>
+                <StyledArrowBackIcon /> <StyledButtonText>{t(getStepLabel(activeStep - 1))}</StyledButtonText>
+              </UpperCaseButton>
+            ) : (
+              <span />
+            )}
+          </div>
+          <StyledRightSideButtonWrapper>
+            <StyledSaveButtonWrapper>
+              <UpperCaseButton variant="outlined" onClick={handleLeaveForm}>
```

Comment:
da er "lagret" teksten borte og "forlat skjerma"-knapp kommet inn. Jeg synes vi beholder "allchangessaved"-logikken i koden likevel

Reply:
Ha en circularprogress som spinner mens man lagrer i stedet?

Reply:
ser for meg at brukeren kommer til å lure på hvorfor det blinker små prikker på skjermen :-)  Lagringen skjer jo som regel umiddelbart